packages/cardano-services/src/cli.ts

```diff
@@ -456,7 +458,17 @@ addOptions(
       for (const queue of queuesArray) if (!isValidQueue(queue)) throw new Error(`Unknown queue name: '${queue}'`);
 
       return queuesArray;
-    }).makeOptionMandatory()
+    }).makeOptionMandatory(),
+    newOption(
+      '--schedules <schedules>',
+      'temp',
```

Comment:
`'temp'` 😄 
It happens... that's why we perform code review 🤝

Reply:
Thanks! Fixed in c91247ca2